docs/develprocess.md

```diff
@@ -108,6 +108,57 @@ Our policy is to upstream everything if possible. However, there are some except
 ### How to help at development
 It all depends on your skills and areas of interest so it's hard to tell specifically in advance. Having a look at the open github issues and pick one (<https://github.com/xcp-ng/xcp/issues>) could definitely help. Else, maybe there's a specific topic that you would want to help improve. Even if you don't know where to start, just come and talk with us (see "Where discussion happens" above).
 
+## Tags, maintenance branches in our code repositories
+
+Objectives of the branch and tag naming conventions:
+* always know how to name tags and maintenance branches depending on the situation
+* easily identify maintenance branches for a given release of XCP-ng, based on their name
+* know what branch to develop the next version on
+* for our tags and branches that have been developed from upstream branches or tags, document the upstream branch names through our branch and tag naming
+
+First question to ask ourselves: **who is the upstream for the software**?
+
+### 1. We are upstream
+
+Example: `xcp-emu-manager`
+* Tags: `vMAJOR.MINOR.PATCH` (`v1.1.2`, `v1.2.0`...)
+* Maintenance branch if needed: `VERSION-XCPNGVERSION}`.
+  * We don't need to create the maintenance branch in advance. Not all software gets hotfixes.
+  * `VERSION` is the version we branched from:
+    * If possible `MAJOR.MINOR` (`1.1`, `1.2`)
+    * ... Unless we tagged the project in a way that would make this ambiguous. In that case, `MAJOR.MINOR.PATCH` (`1.1.1`, `1.1.2`...).
+  * `XCPNGVERSION` is the two-digit version of XCP-ng: `8.1`, `8.2`...
+  * Examples: `1.1.2-8.0`, `1.2-8.2`... 
+* Next release developed on: `master`
+
+* Tag: `v1.2.0`
+* Maintenance branch: `maint-1.2-8.2`
+
+If for any reason we decide to release a newer version of the software as a maintenance update, then:
+* We stop updating the existing maintenance branch
+* Further hotfixes would come from a new maintenance branch created from the appropriate tag.
+
+Special case: VERSION and XCPNGVERSION are always the same (example: `xcp-ng-release`). Then:
+* Tags: `vXCPNGVERSIONFULL` (`v8.2.0`)
+* Maintenance branch if needed: `XCPNGVERSION` (`8.2`)
```

Comment:
Just to be sure, it means this branch is used after `8.2` release and vould contain tags like `v8.2.1` etc?

Reply:
Yes